examples/search-json.js

```diff
@@ -41,50 +45,100 @@ async function searchJSON() {
     client.json.set('noderedis:users:1', '$', {
       name: 'Alice',
       age: 32,
-      coins: 100
+      coins: 100,
+      email: 'alice@nonexist.com'
     }),
     client.json.set('noderedis:users:2', '$', {
       name: 'Bob',
       age: 23,
-      coins: 15
+      coins: 15,
+      email: 'bob@somewhere.gov'
     })
   ]);
 
   // Search all users under 30
   console.log('Users under 30 years old:');
   console.log(
     // https://oss.redis.com/redisearch/Commands/#ftsearch
-    await client.ft.search('idx:users', '@age:[0 30]')
+    JSON.stringify(
+      await client.ft.search('idx:users', '@age:[0 30]'), 
+      null, 
+      2
+    )
   );
   // {
-  //   total: 1,
-  //   documents: [ { id: 'noderedis:users:2', value: [Object] } ]
+  //   "total": 1,
+  //   "documents": [
+  //     {
+  //       "id": "noderedis:users:2",
+  //       "value": {
+  //         "name": "Bob",
+  //         "age": 23,
+  //         "coins": 15,
+  //         "email": "bob@somewhere.gov"
+  //       }
+  //     }
+  //   ]
+  // }
+
+  // Find a user by email - note we need to escape . and @ characters
+  // in the email address.  This applies for other punctuation too.
+  // https://oss.redis.com/redisearch/Tags/#including_punctuation_in_tags
+  console.log('Users with email "bob@somewhere.gov":');
+  const emailAddress = 'bob@somewhere.gov'.replace(/\./g, '\\.').replace(/\@/g, '\\@');
```

Comment:
`.replace(/[.@]/g, '\\$&')`

Reply:
@leibale thanks that's cleaner, I've tested it locally and incorporated.